pkg/config/setup/config.go

```diff
@@ -671,6 +671,8 @@ func InitConfig(config pkgconfigmodel.Setup) {
 
 	// Autoscaling product
 	config.BindEnvAndSetDefault("autoscaling.workload.enabled", false)
+	config.BindEnvAndSetDefault("autoscaling.failover.enabled", false, "DD_AUTOSCALING_FAILOVER_ENABLED")
+	config.BindEnv("autoscaling.failover.metrics", "DD_AUTOSCALING_FAILOVER_METRICS")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is there a particular reason to specify explicitly the environment variables ? They should be used out of the box